package.json

```diff
@@ -30,15 +31,15 @@
   },
   "homepage": "https://github.com/nos/api-functions#readme",
   "dependencies": {
-    "prop-types": "15.6.1"
+    "prop-types": "15.6.2"
   },
   "devDependencies": {
     "@babel/cli": "^7.0.0-beta.46",
```

Comment:
Could you update babel aswell to 7 :P There should be no changes in usage

Reply:
I'd hold off updating babel and do that in a seperate PR. Beta to RC to final might still involve things that break and I want to be sure 💪